components/apimgt/org.wso2.carbon.apimgt.rest.api.gateway.v1/src/main/resources/gateway-api.yaml

```diff
@@ -71,28 +71,32 @@ paths:
       security:
         - OAuth2Security:
             - apim:api_publish
-      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" -X POST \"https://127.0.0.1:9443/api/am/gateway/v1/deploy-api?apiId=890a4f4d-09eb-4877-a323-57f6ce2ed79b&apiName=PizzaAPI&label=Food\""
+      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" -X POST \"https://127.0.0.1:9443/api/am/gateway/v1/deploy-api?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com\""
       x-wso2-request: |
-        POST https://127.0.0.1:9443/api/am/gateway/v1/deploy-api?apiId=890a4f4d-09eb-4877-a323-57f6ce2ed79b&apiName=PizzaAPI&label=Food
+        POST https://127.0.0.1:9443/api/am/gateway/v1/deploy-api?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com
```

Comment:
Shall we change this resource to redeploy?

Reply:
https://github.com/wso2/carbon-apimgt/pull/9009/commits/b99a273017722380eaa25c62f86587c522498b9a#diff-3628fb185e745131f3d5bb8e72370524R23

Reply:
https://github.com/wso2/carbon-apimgt/pull/9009/commits/7bc5f9c2d7bdf813d244a41d3a8c078bde6e54c8